src/audio/mic_privacy_manager/mic_privacy_manager_intel.c

```diff
@@ -84,6 +102,9 @@ int mic_privacy_manager_init(void)
 		LOG_INF("mic_privacy init FW_MANAGED mode");
 		mic_privacy_api->set_fw_managed_mode(true);
 		enable_fw_managed_irq(true);
+	} else if (mic_privacy_policy == MIC_PRIVACY_HW_MANAGED) {
+		LOG_INF("mic_privacy init HW_MANAGED mode");
+		mic_privacy_api->enable_dmic_irq(true, handle_dmic_irq);
```

Comment:
Why we need DMIC IRQ at init when DMIC is OFF anyway?
This probably leads to panic at boot

Reply:
That was indeed not needed here.